docs/dev/04-public-api.md

```diff
@@ -111,6 +111,22 @@ runner.run({port: 9876}, function(exitCode) {
 })
 ```
 
+## karma.stopper
+
+### **stopper.stop(options, [callback=process.exit])**
+
+This function will signal a running server to stop.  Equivalent of `karma stop`.  
```

Comment:
The `detached` setting is currently only working when running the server from CLI, since it's basically just calling itself in a new detached process. 

So yes and no: The stop command will stop the server regardless of the `detached` setting and the server will not run detached when starting it programically. If this is a priority I'll certainly look into it.